vendor.mod

```diff
@@ -13,7 +13,7 @@ require (
 	github.com/distribution/reference v0.6.0
 	github.com/docker/cli-docs-tool v0.9.0
 	github.com/docker/distribution v2.8.3+incompatible
-	github.com/docker/docker v28.0.0-rc.3.0.20250219215346-af898abe4466+incompatible // (v28.0.0-dev)
+	github.com/docker/docker v28.0.1+incompatible
```

Comment:
Something tells me I forgot to push a branch to update it to v28.0.0 😂 (I know I did for buildx; probably forgot for cli)